tests/DispatcherTest.php

```diff
@@ -54,4 +54,21 @@ public function testDispatchMethodNotAllowedRequest()
         $dispatcher = new Dispatcher($this->router);
         $dispatcher->dispatch($request);
     }
+
+    public function testDispatchGot500Exception()
+    {
+        $this->setExpectedException('Exception');
+
+        $router = $this->prophesize('FastRoute\Dispatcher');
+        $request = (new Request())
+        ->withUri(new Uri('/'))
+        ->withMethod('POST');
+
+        $router->dispatch('POST', '/')->willReturn([
+            0 => 3
+        ])->shouldBeCalled();
+
+        $dispatcher = new Dispatcher($router->reveal());
+        $dispatcher->dispatch($request);
```

Comment:
this tests 500 Exception error, apply `shouldBeCalled` to `MethodProphecy` to prove it actually called.